universal-login-relayer/src/integration/ethereum/DeploymentBalanceChecker.ts

```diff
@@ -0,0 +1,13 @@
+import {BalanceChecker, ensure} from '@unilogin/commons'
+import {NotEnoughBalance} from '../../core/utils/errors';
+import {BigNumber} from 'ethers/utils';
+
+export class DeploymentBalanceChecker{
+  constructor(private balanceChecker: BalanceChecker) {}
+
+  async validateBalance(contractAddress: string, tokenAddress: string, gasPriceInToken: BigNumber){
```

Comment:
I thought about it and I think that we should rename `gasPriceInToken` to `gasInToken` because this variable will be taken from calculation: `gasPriceInToken * gasUsed`